### PR TITLE
Request to merge feature popup window

### PR DESCRIPTION
**Pull request by Kingstalux**
I implemented a popup window that includes the project details in both desktop and mobile views.
 In order to do that, I needed to store the information about my projects in a JavaScript object so that when a user loads the page, the projects will be created dynamically by javascript.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Kingstalux, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -1,7 +1,533 @@
...
+  },
+
+  {
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#3',
+    'link to source': '#4',
+  },
```
Comment (mke2111) [inline script.js]: - [x] You are required to use the same array of objects for all your versions (desktop and mobile) for the page. Each object should include at least name, description, image, tech, links. Currently, you have two variables for your array of objects and each has different keys. Give this a fix.

File: script.js [hunk 2]
```diff
@@ -1,7 +1,533 @@
...
+div23.appendChild(div24);
+const button10 = document.createElement('button');
+button10.type = 'button';
+button10.className = 'button1';
+button10.innerText = p1.button;
+button10.onclick = function () { myFunction4(); };
+div23.appendChild(button10);
+
+// rendering mobile projects
+function renderProject(p) {
```
Comment (mke2111) [inline script.js:142]: - [x] You page becomes totally unresponsive once I click See project. Give it a fix.

Screenshot from 2021-09-03 12-43-46

File: script.js [hunk 3]
```diff
@@ -1,7 +1,533 @@
...
+  tech4.appendChild(li);
+}
+div8.appendChild(tech4);
+div7.appendChild(div8);
+const button4 = document.createElement('button');
+button4.type = 'button';
+button4.className = 'button2';
+button4.innerText = p4.button;
+button4.onclick = function () { myFunction2(); };
+div7.appendChild(button4);
```
Comment (mke2111) [inline script.js]: - [x] You are basically assigning each project to HTML structure through DOM one by one. You can still do it in Javascript dynamically. Read more about how you can use iterations of the array of objects.

File: script.js (hunk 1 above)
Comment (Kingstalux) [reply script.js]: okay I'll do that

File: script.js (hunk 2 above)
Comment (Kingstalux) [reply script.js:142]: Okay, once you click on the 'see project' button, the popup menu appears and the scroll is disabled until you click on the "x" icon, then everything goes to normal.

File: script.js (hunk 3 above)
Comment (Kingstalux) [reply script.js]: Okay I've use the iterations of the array of objects to repeat the desktop cards.

File: script.js (hunk 2 above)
Comment (mke2111) [reply script.js:142]: Yes, But it scrolls all the way up to the top of the page and like you said, scroll is disabled. I had to zoom out from 100% to 30% to be able to see your popup window.

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Kingstalux,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 4]
```diff
@@ -1,7 +1,350 @@
...
+div23.appendChild(div24);
+const button10 = document.createElement('button');
+button10.type = 'button';
+button10.className = 'button1';
+button10.innerText = p1.button;
+button10.onclick = function () { myFunction4(); };
+div23.appendChild(button10);
+
+// rendering mobile projects
+function renderProject(p) {
```
Comment (mke2111) [inline script.js:142]: - [x] This issue hasn't been solved yet. The popup leads to the page to be stuck on the top part with the scroll disabled, one wouldn't access it. Please ensure to follow the figma design spec for this popup implementation.

File: script.js [hunk 5]
```diff
@@ -1,7 +1,350 @@
...
+
+  {
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#3',
+    'link to source': '#4',
+    class: "projects_art"
+  },
```
Comment (mke2111) [inline script.js]: - [x] As previously requested, each object should have at least a name, description, image, tech, and link keys. All of your objects except the first are missing some keys.

Comment (Kingstalux) [reply script.js]: Most of the projects have background images that have been applied to the CSS.

File: script.js (hunk 4 above)
Comment (Kingstalux) [reply script.js:142]: okay, I've enabled the scroll. Hope this resolves the issue.

File: script.js (hunk 2 above)
Comment (Kingstalux) [reply script.js:142]: zoom at 30% png
This is my zoom at 30% and the popup remains at the top with no need to scroll before seeing it so any issue might be coming from your end.

File: script.js (hunk 4 above)
Comment (mke2111) [reply script.js:142]: Not entirely, The popup needs to come to display from the same section of the project. The same way you click on a dropdown and it gets you the display within that same section.

Comment (Kingstalux) [reply script.js:142]: Please can you just jump on a quick zoom call and explain clearly what you're saying.

File: script.js (hunk 5 above)
Comment (mke2111) [reply script.js]: Remember, this portfolio will have different projects that you'll create and they'll definitely have different images

File: index.html [hunk 6]
```diff
@@ -23,9 +23,13 @@
 </head>
 <body>
    <section class="body">
-    <section class="sect">
 
-      
+    <section id="popup_mobile" class="multi1"></section>
```
Comment (SajjadAhmad14) [inline index.html:27]: [ OPTIONAL ] Cross icon does not show and it's broken on mobile view. Please see the screenshot below and fix that.
cross

File: script.js (hunk 5 above)
Comment (Kingstalux) [reply script.js]: okay, I added the background images also.

File: index.html [hunk 7]
```diff
@@ -23,9 +23,13 @@
...
+
```
Comment (SajjadAhmad14) [inline index.html:28]: [ OPTIONAL ] Icons inside pop-up buttons are not aligned vertically with text. Kindly see the screenshot below and fix the alignment of the icons.
icons

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Kingsely Ngu, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 6 above)
Comment (Kingstalux) [reply index.html:27]: okay let me check that.

File: index.html (hunk 7 above)
Comment (Kingstalux) [reply index.html:28]: I'll get right on that.

**Pull request merged**